src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue';
+  getArea(): number,
```

Comment:
create separate types for shapes and colors

```suggestion
  shape: Shape,
  color: Color;
  getArea(): number,
```